gfx.cpp

```diff
@@ -4363,7 +4440,7 @@ class GfxInternal
         bound_depth_stencil_target_ = {};
     }
 
-    inline bool isInterop() const { return (swap_chain_ != nullptr ? false : true); }
+    inline bool isInterop() const { return is_interop_; }
```

Comment:
ah I guess the boolean is added because otherwise a headless context would be treated as an interop one with this check, please confirm.

Reply:
True, that's the reason boolean was added. And by checking if swapchain is null we can determine whether windowless (swapchain-less) mode is used.